### PR TITLE
Update docs for 1.7.1

### PR DESCRIPTION
**Pull request by mikesplain**

### CODE REVIEW

File: docs/examples/basic-requirements.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ As root (either ssh directly to root, local root console, or by using "sudo su -
 ```bash
 cd ~
 curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
-curl -LO https://github.com/kubernetes/kops/releases/download/1.7.0/kops-linux-amd64
+curl -LO https://github.com/kubernetes/kops/releases/download/1.7.1/kops-linux-amd64
```
Comment (chrislovecnm) [inline docs/examples/basic-requirements.md]: Can we do that curl call for the latest release?  I know there is a curl command for that.

Comment (mikesplain) [reply docs/examples/basic-requirements.md]: @chrislovecnm I'll look at that, although "latest" right now is technically `1.8.0-beta.1`, should that me move to `pre-release`?

Comment (mikesplain) [reply docs/examples/basic-requirements.md]: Also @chrislovecnm google seems to say it's only possible if we do something like this
```
curl -LO https://github.com/kubernetes/kops/releases/download/$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | grep tag_name | cut -d '"' -f 4)/kops-linux-amd64
```

Let me know if that's acceptable!

**Comment by mikesplain** [comment]
Updated and cleaned up for consistency.  

As a side note, I'm adding `--devel` for [our homebrew forumla](https://github.com/Homebrew/homebrew-core/pull/20464) to make it easier for folks to use our beta releases.  I'll follow up with an additional PR once that's merged.

**Comment by justinsb** [comment]
/lgtm

For the brew - that sounds good.  I think the big problem we had in the past was we were maybe pulling from head, or users were choosing to do so.  I like the idea of giving a --devel flag to give a middle-ground :-)  (And we're not building pulling from master by default, are we?)

**Comment by mikesplain** [comment]
@justinsb Thanks!

Yep, that happened to me in the past.  Nope, we link to the tagged source in brew, then it's bottled up for actual releases to make installs faster.  `--devel` and `--HEAD` allow to build dev branch or actual git head. I'll add to docs today.

**Pull request merged**